### DISCUSSION TITLE
Scrum Prep Milestone 2- Week 1

### DISCUSSION THREAD

**Discussion by PedroRodz** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#79] [Scrum Preps]
Weekly Retrospective
27 sept- 4 oct

**Comment by PedroRodz** [comment]
Opened a lecture topic task. Joined a new group. Discussed the feature with the team.

**Comment by christianmedina-dev** [comment]
Joined the Log In Team. Discussed Milestone 2 next steps

**Comment by Carloscaban** [comment]
Joined the login team, discussed and assigned tasks.

**Comment by keisanabria** [comment]
Joined Documentation team and attended meetings

**Comment by J3SSY-ANDU** [comment]
Joined Documentation and College Plan team. Discussed future features with the college planning team.

**Comment by carlospepin23** [comment]
- Joined the College Planning team.
- Discussed the feature with the team.
- Updated database scheme to include curriculums.

**Comment by cristianMarcial** [comment]
Joined curriculum team and defined tasks
